pages/examples/projects.md

```diff
@@ -27,6 +27,20 @@ These are some awesome projects built using resin.io and are a lot more fully fe
 A [Firebase][firebase]-backed Digital Temperature Logger, written in node.js, allowing you to connect
 devices with multiple temperature sensors to a central cloud-based datastore.
 
+### Audio Stock Ticker
+
+[Repository][audio-stock-ticker-link]
+
+This is a simple node.js project built on resin.io. It should work on all versions of the Raspberry Pi.
+
+The audio stock ticker will verbally announce a list of your favourite stocks every couple of minutes or hours, depending on how you configure it.
+
+### Process and Device Health Monitor
+
+[Repository][pm2-repo-link]
+
+This is a simple example project for Resin.io which demonstrates how to use pm2 on your device to keep a process running forever and monitor your devices health. For the full story check our [blog post][pm2-blog-link] about it.
```

Comment:
haha, I copied that directly from your description on the repo:
https://github.com/resin-io-projects/resin-pm2-monitor
I will fix it now. thanks